mantidproject/mantid
# Failing to find added files

**Description of work.**

This PR adds the instrument name to the front of files added in the new SANS GUI. It also improves the filefinder so that files which do not begin with the instrument name have a chance to be found. 

**Report to:** Diego. 

**To test:**
 * Get this data, add the folder to the search directories and set the folder as the default save directory.
[loqdemo.zip](https://github.com/mantidproject/mantid/files/2401851/loqdemo.zip)
 * In the new SANS GUI navigate to the add tab. Elect to add LOQ74044 to itself.
* Check that the suggested name is 'LOQ74044-add'
* Change the suggested name to '74044-add'
* in the runs tab type '74044-add' into the Sample scatter box and click reduce, check that the correct file is found. 




Fixes #22378. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
